stdlib/@tests/stubtest_allowlists/py313.txt

```diff
@@ -204,3 +194,7 @@ codecs.namereplace_errors
 codecs.replace_errors
 codecs.strict_errors
 codecs.xmlcharrefreplace_errors
+
+# Runtime dynamically renames kwargs
```

Comment:
Runtime uses a decorator to support the leagcy kwargs: https://github.com/python/cpython/blob/2bac2b86b1486f15038fb246835e04bb1b213cd8/Lib/importlib/resources/_common.py#L4